travis-ci/managers/fuzzit.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -e
+set -x
+set -u
+
+REPO_ROOT=${REPO_ROOT:-$(pwd)}
+
+sudo bash -c "echo 'deb-src http://archive.ubuntu.com/ubuntu/ xenial main restricted universe multiverse' >>/etc/apt/sources.list"
+sudo apt-get update -y
+sudo apt-get build-dep systemd -y
+sudo apt-get install -y ninja-build python3-pip python3-setuptools
+pip3 install meson
+
+cd $REPO_ROOT
+export PATH="$HOME/.local/bin/:$PATH"
+tools/oss-fuzz.sh
+
+export FUZZIT_API_KEY=7c1bd82fe0927ffe1b4bf1e2e86cc812b28dfe08a7080a7bf498e98715884a163402ee37ba95d4b1637247deffcea43e
```

Comment:
Could you add a comment about why we decided to keep it public and why in general given threat modelling :-) it's safe to avoid answering the same questions over and over again?

Reply:
Currently because we want Fuzzit to run sanity jobs on every pull-request we can't use travis secrets as they are not exposed to pull-request and this is why we use this key in exposed form. The key is a create only key so it doesn't have permission to read sensitive data from the Fuzzit project.

Reply:
Sorry. My comment wasn't clear enough. What I was trying to say is that I think the comment should be added to the script itself right before `export FUZZIT_API_KEY`.

Reply:
Got it. will do.